flink-architecture-tests/flink-architecture-tests-base/src/main/java/org/apache/flink/architecture/common/Predicates.java

```diff
@@ -48,7 +47,9 @@ public static DescribedPredicate<JavaClass> areDirectlyAnnotatedWithAtLeastOneOf
             Class<? extends Annotation>... annotations) {
         return Arrays.stream(annotations)
                 .map(CanBeAnnotated.Predicates::annotatedWith)
-                .reduce(DescribedPredicate::or)
+                .reduce(
+                        (canBeAnnotatedDescribedPredicate, other) ->
+                                canBeAnnotatedDescribedPredicate.or(other))
```

Comment:
Did the added static `or` method ruin this? :weary:

Reply:
seems like that here
https://github.com/TNG/ArchUnit/commit/2065e012e26b2913257efc04fc4d4868176b459e#diff-05df2dd6f3ee640302d7633e8d17fb55cee3a3a43c3d72e41e727a0ef8037dc3R191-R209